src/internal/_Set.js

```diff
@@ -0,0 +1,167 @@
+var _contains = require('./_contains');
+
+
+// A simple Set type that honours R.equals semantics
+module.exports = (function() {
+  function _Set() {
+    /* globals Set */
+    this._nativeSet = typeof Set === 'undefined' ? null : new Set();
```

Comment:
Is there a reason to favour this negatory assertion over `typeof Set === 'function'`?